tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -508,9 +509,12 @@ def testee(a: cases.EField) -> cases.EField:
         unstructured_case,
         testee,
         ref=lambda a: np.sum(
-            np.sum(a[unstructured_case.offset_provider["V2E"].table], axis=1)[
-                unstructured_case.offset_provider["E2V"].table
-            ],
+            np.sum(
+                a[unstructured_case.offset_provider["V2E"].table],
+                axis=1,
+                initial=0,
+                where=unstructured_case.offset_provider["V2E"].table > 0,
```

Comment:
Just a question, why `> 0 ` is used here instead of comparing against `SKIP_VALUE` as done everywhere else?

Reply:
oversight